src/components/Question.js

```diff
@@ -19,10 +19,11 @@ const NavigationButton = props =>
 class Question extends Component {
   constructor() {
     super();
-    this.state = { userAnswerIndex: null, answerSubmitted: null, error: false };
+    this.state = { userAnswerIndex: null, answerSubmitted: null, error: false, correctAnswer: null };
   }
 
   handleSubmit(event) {
+    let currentAnswer = this.props.question.answers[this.state.userAnswerIndex];
```

Comment:
I like that you determine the `currentAnswer` via props and can easily check if it is correct via `currentAnswer.isCorrect`.